torch/csrc/distributed/rpc/init.cpp

```diff
@@ -121,7 +121,14 @@ PyObject* rpc_init(PyObject* /* unused */) {
           .def(
               "get_metrics",
               &RpcAgent::getMetrics,
-              py::call_guard<py::gil_scoped_release>());
+              py::call_guard<py::gil_scoped_release>())
+          .def(
+              "_get_timeout",
+              // intentionally not releasing GIL to avoid context switch
+              [](const RpcAgent& self) {
+                return self.getRpcTimeout().count() / float(kToMilliseconds);
```

Comment:
nit: there is also `kSecToMsConversion` which is 1000, does this serve the same purpose?

Reply:
consolidated all conversion const into the one in `rpc_agent.h`